project/plugins.sbt

```diff
@@ -1,6 +1,6 @@
 addSbtPlugin("com.twitter" % "scrooge-sbt-plugin" % "22.1.0")
 
-addSbtPlugin("com.gu" % "sbt-scrooge-typescript" % "2.0.0")
+addSbtPlugin("com.gu" % "sbt-scrooge-typescript" % "3.0.3")
 
```

Comment:
Currently we can't update the `sbt-scrooge-typescript` to `3.0.3` because the `scrooge` version in this library https://github.com/guardian/scrooge-extras/blob/main/build.sbt#L15C5-L15C19 bans some key words like `end`. More details can be found here https://github.com/guardian/content-api-models/pull/231#issuecomment-1900668948

The compilation error for this update is as followed:
```
Caused by: com.twitter.scrooge.frontend.FileParseException: Exception parsing: content/v1.thrift
[error] Caused by: com.twitter.scrooge.frontend.KeywordException: Identifier 'end' is invalid: it is a thrift keyword.
```